<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Client Catcher</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
    background-color: rgb(45, 45, 45);
}

QPushButton {
    background-color: #303030;
    color: white;
    border: 1px solid #7fcde1;
    border-radius: 2px;
    padding: 4px;
    font: 10pt &quot;Arial&quot;;
    height: 20px;
    
    
}

/* Add a hover effect to buttons */
QPushButton:hover {
    background: #ff9d00;
    color: #ffffff
}

/* Add a pressed effect to buttons */
QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
    stop: 0 #ff9d00, stop: 1 #ffb900);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="Top_Bar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(35, 35, 35);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_top">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>52</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
color: white;
font: 20pt &quot;Arial&quot;</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>3</number>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../../Desktop/Caio Cardoso/whiteloggg.PNG</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">font: 18pt Arial;
color: 
qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.306818 rgba(0, 0, 0, 236), stop:1 rgba(255, 255, 255, 255))</string>
            </property>
            <property name="text">
             <string>Client Catcher</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="Content">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_left_menu">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(35, 35, 35);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignTop">
           <widget class="QFrame" name="frame_top_menus">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="Btn_Toggle">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: rgb(35, 35, 35);
    color: rgb(162, 162, 162);
    border: 1px solid rgb(85, 85, 85);
    border-radius: 2px;
    
	margin-left: 20px;
   margin-right: 20px;
    margin-top: 15px;
   margin-bottom: 15px;
    font: 10pt &quot;Arial&quot;;
    
    
    
}

/* Add a hover effect to buttons */
QPushButton:hover {
    border: 1px solid #7fcde1;
    color: #7fcde1
}

/* Add a pressed effect to buttons */
QPushButton:pressed {
    color: #ffffff;
    
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_page_1">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	color: rgb(154, 154, 154);
	background-color: rgb(35, 35, 35);
	border: 0px solid;
}
QPushButton:hover {
     background-color: rgb(113, 113, 113);
     color: #ffffff;
}</string>
               </property>
               <property name="text">
                <string>Load File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_page_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	color: rgb(154, 154, 154);
	background-color: rgb(35, 35, 35);
	border: 0px solid;
}
QPushButton:hover {
     background-color: rgb(113, 113, 113);
     color: #ffffff;
}</string>
               </property>
               <property name="text">
                <string>BI Workspace</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_page_3">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	color: rgb(154, 154, 154);
	background-color: rgb(35, 35, 35);
	border: 0px solid;
}
QPushButton:hover {
     background-color: rgb(113, 113, 113);
     color: #ffffff;
}</string>
               </property>
               <property name="text">
                <string>AE Collaboration</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_page_4">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	color: rgb(154, 154, 154);
	background-color: rgb(35, 35, 35);
	border: 0px solid;
}
QPushButton:hover {
     background-color: rgb(113, 113, 113);
     color: #ffffff;
}</string>
               </property>
               <property name="text">
                <string>Reach Out</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_page_5">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	color: rgb(154, 154, 154);
	background-color: rgb(35, 35, 35);
	border: 0px solid;
}
QPushButton:hover {
     background-color: rgb(113, 113, 113);
     color: #ffffff;
}</string>
               </property>
               <property name="text">
                <string>Profile Info</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_pages">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="page_1">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <property name="spacing">
               <number>9</number>
              </property>
              <property name="leftMargin">
               <number>9</number>
              </property>
              <property name="topMargin">
               <number>9</number>
              </property>
              <property name="rightMargin">
               <number>9</number>
              </property>
              <property name="bottomMargin">
               <number>9</number>
              </property>
              <item alignment="Qt::AlignHCenter">
               <widget class="QPushButton" name="loadButton">
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    background-color: rgb(53, 53, 53);
    color: rgb(125, 125, 125);
    border: 1px solid #7fcde1;
    border-radius: 2px;
    padding: 4px;
    font: 10pt &quot;Arial&quot;;
    height: 80px;
    
    
    
}

/* Add a hover effect to buttons */
QPushButton:hover {
    background: rgb(71, 71, 71);
    font: rgb(204, 204, 204);
    color: #ffffff
}

/* Add a pressed effect to buttons */
QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
    stop: 0 #ff9d00, stop: 1 #ffb900);
}</string>
                </property>
                <property name="text">
                 <string>Open Excel File</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <layout class="QVBoxLayout" name="verticalLayout_6"/>
            </widget>
            <widget class="QWidget" name="page_3">
             <layout class="QVBoxLayout" name="verticalLayoutAE"/>
            </widget>
            <widget class="QWidget" name="page_4">
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Desktop/test3.qrc"/>
 </resources>
 <connections/>
</ui>
